Improve readme

uscholdm opened the pull request:
Fixes #10. Fixes #13 

- Greatly improved the README file and tweaked a few annotations.
- Fixed an import statement.
- Only importing versionIRIs. 
- Added example annotations

rjyounes commented on README.md:
```diff
@@ -1,2 +1,11 @@
+1. the version of a disease ontology being at least 3.1.0 and less than 4.0.0. 
+2. the version of gist core being between at least 12.1.0 and less than 13.0.0.
+
+The need for such specifying such dependencies arises when ontologies use terms from imported ontologies.
```
```suggestion
The need for specifying such dependencies arises when ontologies use terms from imported ontologies.
```

rjyounes commented on README.md:
```diff
@@ -1,2 +1,11 @@
+
+The need for such specifying such dependencies arises when ontologies use terms from imported ontologies.
+
+Three SPARQL queries are provided of perform the following tasks:
```
```suggestion
Three SPARQL queries are provided to perform the following tasks:
```

rjyounes commented on examples/examplesVersionDependency.ttl:
```diff
@@ -25,13 +25,13 @@
 	skos:definition "A hierarchy of diseases."^^xsd:string ;
 	skos:prefLabel "Disease Taxonomy"^^xsd:string ;
-	skos:scopeNote "The semantic version number for the disease taxonomy is 6.3.0. This version depends on the version of the disease ontology being between 3.1.0 and 4.0.0. It also depends on the version of gist core being between 12.1.0 and 13.0.0."^^xsd:string ;
+	skos:scopeNote "The semantic version number for the disease taxonomy is 6.3.0. This version depends on the version of the imported disease ontology being at least 3.1.0 and less than 4.0.0. It also depends on the version of gist core being at least 12.1.0 and less than 13.0.0."^^xsd:string ;
```
Do we really want such a note? Shouldn't the dependency assertions be sufficient? I don't think we should suggest to users that they should include this.

uscholdm replied:
Good point, it does not belong here. I will move this to a skos:example on the versioning ontology.

rjyounes replied:
I'm confused - you added the example there but it looks like it didn't get deleted here.

uscholdm replied:
It is now removed from both from the examples file. I put it in a `skos:example` annotation in the versioning ontology itself.

rjyounes commented on examples/examplesVersionDependency.ttl:
```diff
@@ -66,6 +66,7 @@
 
 <https://w3id.org/semanticarts/ontology/gistCore>
 	a owl:Ontology ;
+	owl:versionIRI <https://w3id.org/semanticarts/ontology/gistCore12.2.2> ;
```
Your other example on line 28 uses the unversioned import. Do you want to show that either are acceptable? Should we follow the best practice of not versioning the import IRI if there's a dependency assertion?

uscholdm replied:
I'm not 100% sure what we decided at today;s gist meeting.

rjyounes replied:
We decided that the best practice is to use a versioned IRI in the import and that it should be within the range specified in the dependencies.

rjyounes reviewed (changes requested):
So let's go back to what you had previously on line 28, given the best practice decided on today.

rjyounes commented on examples/examplesVersionDependency.ttl:
```diff
@@ -25,13 +25,13 @@
 	owl:imports
 		<https://ontologies.exco.com/exeoDiseases> ,
-		<https://w3id.org/semanticarts/ontology/gistCore12.1.0> ,
+		<https://w3id.org/semanticarts/ontology/gistCore> ,
```
We said we should use versioned IRIs for imports.

rjyounes reviewed (approved):
OK to merge.

uscholdm commented:
Fixes #10. Fixes #13 

- Greatly improved the README file and tweaked a few annotations.
- Fixed an import statement.
- Only importing versionIRIs. 
- Added example annotations
